using-datapower-in-kubernetes/getting-started-datapower-in-gke.md

```diff
@@ -0,0 +1,350 @@
+* __Compute Engine API__
+* __Container Engine API__
+
+![Google API Console](media/gcp_api_manager.png)And familiarize yourself with the Google Cloud Shell which we will use to manage our
+Google Cloud Platform resources. You can find a getting started for the Google Cloud Shell 
+[here](https://cloud.google.com/shell/docs/quickstart "Google Cloud Shell Getting Started").
+Once ready, head over to your project page in the Google Cloud Platform web console and
+start a Google Cloud Shell session.
+
+![Activate Google Cloud shell](media/gcp_activate_gcs.png)
+
+To avoid having to specify the --zone flag when required, you can set the region with:
+    
+```
+$ gcloud config set compute/zone <zone>
+```
+To see valid choices for `<zone>`, run:
+```
+$ gcloud compute zones list
+```
+
+### Setting Up a Kubernetes cluster with GKE (Optional)
+
+GKE provides a hosted Kubernetes service that can be customized. We will leverage Kubernetes to handle the complexity 
+associated with orchestrating application containers.
+
+To provision a GKE cluster, open up your project in Google Cloud Platform and start an instance of the Google Cloud Shell.
+Next, run the following command to create a GKE cluster of 3 nodes for use in this tutorial:
+
+```
+$ gcloud container clusters create k8s-cluster
+
+kubeconfig entry generated for k8s-cluster.
+NAME          ZONE        MASTER_VERSION  MASTER_IP       MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
+k8s-cluster  us-east1-b  1.4.7           104.196.69.190  n1-standard-1  1.4.7         3          RUNNING
+```
+
+This might take a couple of minutes.
+
+With our cluster up and running, we are now ready to run containers. 
+
+### Get the Source Code
+
+```
+$ git clone https://github.com/ibm-datapower/datapower-tutorials.git
+$ cd datapower-tutorials/using-datapower-in-kubernetes/
+```
+
+
+
+
+
+##  Introduction to Kubernetes
+
+
+
+
+###  Pods
+
+In Kubernetes, all containers run in what's called a pod. A pod represents a logical application composed of co-located 
+and co-scheduled containers, shared storage, and certain shared namespaces. Pods, not containers, are the smallest deployable artifact in Kubernetes.
+
+To run a pod directly, we can, for example, run a DataPower container with:
+
+```
+$ kubectl run --stdin --tty datapower --image=ibmcom/datapower:latest --env="DATAPOWER_ACCEPT_LICENSE=true" --env="DATAPOWER_INTERACTIVE=true"
+
+```
+
+This will automatically create a default Kubernetes _deployment_ , which is a Kubernetes construct that lets you declare the desired state of
+your application and is used to update your pods or replica sets to try to match the current state of your application to that of the desired state.
+You can specify deployments, pods, services, replica sets, etc. declaratively in Kubernetes by writing a
+corresponding configuration file.
+
+In a separate Google Cloud Shell session, we can run `$ kubectl describe deployments` to show the details
```

Comment:
Unsure if I should address this, I mentioned how to set the compute/zone in the beginning of the tutorial